Snowflake.Data/Core/IRestRequest.cs

```diff
@@ -114,6 +114,15 @@ class AuthnRequestClientEnv
 
         [JsonProperty(PropertyName = "OS_VERSION")]

```

Comment:
What is the OS version look like?

Reply:
This is how OS version looks like `ClientEnv: { APPLICATION: dotnet, OS_VERSION: Microsoft Windows NT 6.2.9200.0, NET_RUNTIME: .NET Core 4.6.26212.01 }`

Reply:
This is how JDBC client environment build:
```
      Map<String, Object> clientEnv = new HashMap<String, Object>();
      clientEnv.put("OS", System.getProperty("os.name"));
      clientEnv.put("OS_VERSION", System.getProperty("os.version"));
      clientEnv.put("JAVA_VERSION", System.getProperty("java.version"));
      clientEnv.put("JAVA_RUNTIME", System.getProperty("java.runtime.name"));
      clientEnv.put("JAVA_VM", System.getProperty("java.vm.name"));
```
JDBC also have some java specific attribute.